R/as.network.numeric.R

```diff
@@ -65,6 +65,25 @@ as.network.numeric<-function(x,
     ignore.eval = TRUE, names.eval = NULL,
     edge.check = FALSE,
     density=NULL, init=NULL, numedges=NULL, ...){
+  # Producing an informative error for each of the following invalid or unsupported inputs
+  if(loops == TRUE)
+    stop("Loop can't be contained in your random network")
+  if(multiple == TRUE)
+    stop("Multiple edges can't be contained in your random network")
+  if(hyper == TRUE)
+    stop("Hyper edges can't be contained in your random network")
+  if((missing(density)!=TRUE) && (density <0 || density>1))
+    stop("Density of graph can't be either negative or greater then 1")
+  if(is.null(numedges)!=TRUE && is.integer(numedges)!=TRUE)
+    stop("The number of edges must be an integer value")
+  if(missing(directed)!=TRUE && directed == TRUE && bipartite != FALSE)
+    stop("Directed bipartite network is not allowed to be generated")
+  if(missing(bipartite) && missing(directed))
+    stop("It requires more arguments, specify whether the network is bipartite or directed")
+  if(bipartite!=FALSE && missing(directed)){
+    directed <- FALSE
+    warning("If the netword is set to be bipartite, it considers as an undirected graph. This behaviour may change in the future")
```

Comment:
Reword as "Bipartite network specified; assuming undirected. This behavior may change in the future."